### PR TITLE
Minor Mission Wording Changes

### PR DESCRIPTION
**Pull request by Evan-A-Wolf**
There were just a couple of words/sentences that I felt worked better this way. They were mostly edited to have better conational consistency or to have more consistent implications. There are just three, but I felt like they should be changed. They've been annoying me for a while now.

First change was to change it from an order to a teaching moment. Considering the way James acts, I think that this a bit more like what would happen.
Second change is just for a little bit of chronological consistency. You can't really be taking payment from somebody if they're currently being pulled away and talked to. If you say after it puts it together a bit more.
Third change is for more connotation consistency. Walk is very casual while march is very uptight. Pass is more neutral and doesn't contradict like walk and march does.
Fourth change should be on, not of.

### CODE REVIEW

**Comment by Evan-A-Wolf** [comment]
Also, not sure whether or not I did it right. First time doing this. Usually I'm in charge of the GitHub project, so if I did it wrong, I'm sorry.

File: data/intro missions.txt [hunk 1]
```diff
@@ -135,14 +135,14 @@ mission "Intro [2 Transport]"
 				`	"Sure!"`
 				`	"No thanks, I think I'll strike off on my own now."`
 					decline
-			`	"Great," he says. "This is Chuck, and Sarah, and their son Carl. Let's help them carry their luggage to your ship." As you begin walking back, slightly ahead of them, James whispers to you, "Tourists are always a good way to make money. They're on vacation, so they don't mind paying a bit extra. But no point in your cargo space going to waste, so take a look at the job board and see if there are any delivery missions we can run at the same time."`
+			`	"Great," he says. "This is Chuck, and Sarah, and their son Carl. Let's help them carry their luggage to your ship." As you begin walking back, slightly ahead of them, James whispers to you, "Tourists are always a good way to make money. They're on vacation, so they don't mind paying a bit extra. But no point in your cargo space going to waste, you should take a look at the job board and see if there are any delivery missions we can run at the same time."`
```
Comment (Pointedstick) [inline data/intro missions.txt]: This change makes the sentence grammatically incorrect by turning it into a [comma splice](https://en.wikipedia.org/wiki/Comma_splice).

**Comment by Evan-A-Wolf** [comment]
Yeah, I see what you mean. It's technically not incorrect. If you're writing in a earlier style of English that is, or if you're trying to convey an informal tone. Hint hint nudge nudge that's what I was trying to do.  Still, I think I made a change that fixes that. Not sure though, I've never used GitHub this way before.

**Comment by Evan-A-Wolf** [comment]
sweet. Thanks. And were there any problems with how I did it? Like I said, I'm still learning how to use GitHub.

**Comment by Pointedstick** [comment]
Nope, you did great!

**Comment by endless-sky** [comment]
In general, I don't see a point in making wording changes simply based on different opinions of what sounds good (i.e. not to fix grammatical errors, typos, or unclear or clunky phrasing). None of these changes are obvious improvements, and the middle one makes two paragraphs in a row start with "After," which is a bit awkward.

**Pull request closed without merging**